src/TierRegistry.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
We can use MIT here like the other contracts (the storage registry had a different license ID because of one of its dependencies)